docs/index.rst

```diff
@@ -1401,6 +1401,9 @@ Process class
     id and thread CPU times (user/system). On OpenBSD this method requires
     root privileges.
 
+    .. note::
+      (AIX) This function is not available on older (end of service) versions.
+
```

Comment:
I wouldn't bother signaling this in the doc. AIX is already rarely used. An EOS release is even more rare.